applications/salk-portal/src/components/DensityMap.tsx

```diff
@@ -99,63 +101,154 @@ const RadioButton = ({onChange, isChecked, label}) => {
 }
 
 const DensityMap = (props: {
-    experimentId: string,
     subdivisions: string[], activePopulations: Population[],
-    selectedAtlas: AtlasChoice, selectedValue: string, onChange: (value: string) => void
+    selectedAtlas: AtlasChoice,
+    showProbabilityMap: boolean, showNeuronalLocations: boolean,
 }) => {
     const api = workspaceService.getApi()
-    const {experimentId, activePopulations, selectedAtlas, onChange} = props
-    const [selectedValue, setSelectedValue] = React.useState(props.selectedValue);
-    const [densityRequest, setDensityRequest] = React.useState({
-        loading: false,
-        data: null,
-        state: null
-    });
+    const {activePopulations, selectedAtlas, showProbabilityMap, showNeuronalLocations} = props
+    const activePopulationsColorMap = activePopulations.reduce((acc, pop) => {
+        return {...acc, [pop.id]: pop.color}
+    }, {})
+    // FIXME: useEffect was detecting activePopulations changes although the object content wasn't changing
+    // Line below is a workaround to fix the issue
+    const activePopulationIds = activePopulations.map(pop => `${pop.id}_${pop.color}_${pop.opacity}`).sort().toString()
```

Comment:
This should be rethinked. 
Current implementation is a workaround and not perfect (if cells changes it won't get updated f.e)